parlai/crowdsourcing/tasks/turn_annotations_static/launch_config.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from parlai_internal.mturk.block_list import WORKER_BLOCK_LIST
+
+"""
+
+Use this file to put all the arguments (an alternative to launching them from the command line as params)
+
+"""
+
+
+# Run the below to register the requester
+# mephisto register mturk_sandbox --name:noahturkproject.1038@gmail.com
+# --access-key-id:XXXX --secret-access-key:XXXX
+REQUESTER = 'FIXME'
+PROVIDER = 'mturk_sandbox'
+# This datapath is where the database object goes
+# If not Mephisto data path below then requester register seems to do nothing
+DATAPATH = 'FIXME'
+
+TASK_TITLE = 'Chat with a fellow conversationalist'
+TASK_DESCRIPTION = '''<br>
```

Comment:
This ideally would be a separate file passed in with a flag, but I think this is fine for this PR

Reply:
I've actually made it so there is only one parameter which will be the launch_config. I think separating TASK_DESCRIPTION and the LEFT_PANE_TEXT into separate files will be fine, but I've actually moved away from having command line parameters.